documentation/swagger-authoring-automated-guidelines.md

```diff
@@ -0,0 +1,82 @@
+# Automated Guidelines for Swagger Authoring #
```

Comment:
We haven't categorized the rules between ARM and Data plane, so if you want to keep the title and distinguish in a section or something feel free to do so, but what I can state that the stuff in the current document is for ARM.